wpilibc/athena/src/RobotDrive.cpp

```diff
@@ -7,17 +7,16 @@
 
 #include "RobotDrive.h"
 
-#include <math.h>
+#include <algorithm>
+#include <cmath>
+
 #include "GenericHID.h"
 #include "HAL/HAL.h"
 #include "Joystick.h"
 #include "Talon.h"
 #include "Utility.h"
 #include "WPIErrors.h"
 
-#undef max
```

Comment:
I'm not sure why any of our compilers would provide min() and max() as macros. Doing so wouldn't be standards-compliant afaik.